README.md

```diff
@@ -160,3 +167,12 @@ npm run test -u
 ```bash
 npm run depcheck
 ```
+
```

Comment:
On line 143 (this is further up and I can't select it) we have something like `npm run prelink` but the script `prelink` doesn't seem to be anywhere, what do you think?

Reply:
mm... what do you get if you try to just `npm run lnt`

Reply:
You wanted to say `npm run lint` ?